tests/foreman/virtwho/test_ui_virtwho.py

```diff
@@ -626,3 +626,68 @@ def test_positive_virtwho_manager_role(session, test_name, form_data):
         # Delete the created user
         session.user.delete(username)
         assert not session.user.search(username)
+
+
+def test_positive_overview_label_name(form_data, session):
+    """Verify the label name on virt-who config Overview Page.
+
+    :id: 68422042-648c-4652-a52b-5bf42462d2ae
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
+    """
+    name = gen_string('alpha')
+    form_data['name'] = name
+    hypervisor_type = form_data['hypervisor_type']
+    http_proxy = 'test.rexample.com:3128'
+    no_proxy = 'test.satellite.com'
+    regex = '.*redhat.com'
+    whitelist = {
+        'filtering': 'Whitelist',
+        'filtering_content.filter_hosts': regex}
+    blacklist = {
+        'filtering': 'Blacklist',
+        'filtering_content.exclude_hosts': regex}
+    if hypervisor_type == 'esx':
+        whitelist['filtering_content.filter_host_parents'] = regex
+        blacklist['filtering_content.exclude_host_parents'] = regex
+    with session:
+        session.virtwho_configure.create(form_data)
+        results = session.virtwho_configure.read(name)
+        fields = {
+            'status_label': 'Status',
+            'hypervisor_type_label': 'Hypervisor Type',
+            'hypervisor_server_label': 'Hypervisor Server',
+            'hypervisor_username_label': 'Hypervisor Username',
+            'interval_label': 'Interval',
+            'satellite_url_label': 'Satellite server FQDN',
+            'hypervisor_id_label': 'Hypervisor ID',
+            'filtering_label': 'Filtering',
+            'debug_label': 'Enable debugging output?'
+        }
+        for key, value in fields.items():
+            assert results['overview'][key] == value
+        session.virtwho_configure.edit(
+            name, dict({'proxy': http_proxy,
+                        'no_proxy': no_proxy},
+                       **whitelist))
+        results = session.virtwho_configure.read(name)
+        fields['filter_hosts_label'] = 'Filter Hosts'
+        if hypervisor_type == 'esx':
+            fields['filter_host_parents_label'] = 'Filter Host Parents'
+        fields['proxy_label'] = 'HTTP Proxy'
+        fields['no_proxy_label'] = 'Ignore Proxy'
+        for key, value in fields.items():
+            assert results['overview'][key] == value
+        session.virtwho_configure.edit(name, blacklist)
+        results = session.virtwho_configure.read(name)
+        del fields['filter_hosts_label']
+        if hypervisor_type == 'esx':
+            del fields['filter_host_parents_label']
+            fields['exclude_host_parents_label'] = 'Exclude Host Parents'
```

Comment:
You have set `exclude_host_parents_label` already in lin 678.

Reply:
nope, lin 678 is `filter_host_parents_label` , not this `exclude_host_parents_label`

Reply:
Ah, you are right. Don't know what I was thinking.